src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -14,16 +14,31 @@
 @Entity(name = "USERS")
 public class User {
 
-    public User (@NotNull String username, @NotNull int status, @NotNull String userKey) {
```

Comment:
Ja uważam, że tu wystarczy kosntruktor z username, status, i user key. expirationTime jest można ustawić w ciele konstruktora (w kolejnych logowaniach już setterem) albo w ogóle setterem. Poza tym on jest tylko do sprawdzenia terminu ważności klucza co moim zdaniem dzieje się po stronie backendu i nie trzeba tego eksportować na zewnątrz. Tak naprawdę, to na moje oko id w konstruktorze też nie potrzeba skoro tworzy je Hiberante. Podobnie niżej z listOf Orders. Moim zdaniem na starcie powinna to być pusta lista, do której za pomocą setterów będziemy dodawać rekordy.

Reply:
Akurat konstruktor z ID przydaje się chyba przy transferowaniu między obiektami, a wersjami dto?

Reply:
właśnie to rozgryzam przy pracy nad ProductCotrollerem i może coś być na rzeczy z tym id, wczesniej wydawało mi się, że w Dto wystarczy, żeby dostarczać dane z backendu do frontendu, ale w drugą stronę też może być przydatny. Chociaż nie do końca jestem przekonany, ale jeszcze to rozkiminam :) może do tego logowania być przydatny... ale resztę podtrzymuję